tests/networkpolicy_test.go

```diff
@@ -52,25 +52,17 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		Expect(err).ToNot(HaveOccurred())
 
 		vmib = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-		_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmib)
+		vmib, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmib)
 		Expect(err).ToNot(HaveOccurred())
 
 		vmic = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
 		vmic.Namespace = tests.NamespaceTestAlternative
-		_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestAlternative).Create(vmic)
-		Expect(err).ToNot(HaveOccurred())
-
-		tests.WaitForSuccessfulVMIStart(vmia)
-		tests.WaitForSuccessfulVMIStart(vmib)
-		tests.WaitForSuccessfulVMIStart(vmic)
-
-		vmia, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmia.Name, &v13.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		vmib, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmib.Name, &v13.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		vmic, err = virtClient.VirtualMachineInstance(tests.NamespaceTestAlternative).Get(vmic.Name, &v13.GetOptions{})
+		vmic, err = virtClient.VirtualMachineInstance(tests.NamespaceTestAlternative).Create(vmic)
 		Expect(err).ToNot(HaveOccurred())
 
+		vmia = tests.WaitUntilVMIReady(vmia, tests.LoggedInCirrosExpecter)
+		vmib = tests.WaitUntilVMIReady(vmib, tests.LoggedInCirrosExpecter)
+		vmic = tests.WaitUntilVMIReadyNamespace(vmic, vmic.Namespace, tests.LoggedInCirrosExpecter)
```

Comment:
I think it's preferable to just create an helper to do this, something like what @phoracek did in https://github.com/kubevirt/kubevirt/blob/master/tests/network/primary_pod_network.go#L102 

I'm also copying that 'pattern' in other tests - see [this discussion](https://github.com/kubevirt/kubevirt/pull/4035#discussion_r478201237) about keeping the helper names coherent.

Reply:
done